server/apps/entry/models.py

```
@@ -73,11 +73,15 @@ def get_pdf(cls, url: str, instance: 'SourcePreview' = None, **kwargs) -> 'Sourc
 
 class OSMName(UUIDAbstractModel, models.Model):
     class OSM_ACCURACY(enum.Enum):
-        ADMIN = 0
-        POINT = 1
+        COUNTRY = 0
+        STATE = 1
+        COUNTY = 2
+        POINT = 3
```

@annshress Don't we also need migration for this?
CC: @thenav56

---

We do. Good catch. @annshress Make sure not to silently ignore this in the future.
But we can skip for now, since we don't have any real data.